src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -571,7 +571,7 @@ static constexpr Property<ExecutionMode> execution_mode{"EXECUTION_MODE_HINT"};
  * might result in better accuracy, but the drawback is worse performance. Group size equal 0 means dynamic
  * quantization optimization is disabled.
  */
-static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization_group_size{
+static constexpr Property<int64_t, PropertyMutability::RW> dynamic_quantization_group_size{
```

Comment:
num_streams is encapsulated with namespace, but you are not suggesting to introduce a new name space for group_size, right? If we do so, it will break API compatibility.. Then it will look like below. I cannot use general word like `Num` because it is not within a namespace. Does this look good?
```

struct GSNum  {
    constexpr GSNum() : gs_num{0} {};

    constexpr GSNum(const uint64_t num_) : gs_num{num_} {}

    constexpr operator uint64_t() const {
        return gs_num;
    }

    uint64_t gs_num = 0;
};

static constexpr GSNum PER_TOKEN{UINT64_MAX};

static constexpr GSNum DISABLED{0};

static constexpr Property<GSNum, PropertyMutability::RW> dynamic_quantization_group_size{
    "DYNAMIC_QUANTIZATION_GROUP_SIZE"};

```

Reply:
Maybe something like this?
```cpp
namespace dynamic_quantization {
struct GroupSize  {
    constexpr GroupSize() : gs_num{0} {};

    constexpr GroupSize(const uint64_t num_) : gs_num{num_} {}

    constexpr operator uint64_t() const {
        return gs_num;
    }

    uint64_t gs_num = 0;
};

static constexpr GroupSize PER_TOKEN{UINT64_MAX};

static constexpr GroupSize DISABLED{0};

static constexpr Property<GroupSize, PropertyMutability::RW> group_size{"DYNAMIC_QUANTIZATION_GROUP_SIZE"};

} // namespace dynamic_quantization

// keep it for compatibility
static constexpr Property<dynamic_quantization::GroupSize, PropertyMutability::RW> dynamic_quantization_group_size{"DYNAMIC_QUANTIZATION_GROUP_SIZE"}; 

// ...

  core.set_property(ov::hint::dynamic_quantization_group_size(32));
  core.set_property(ov::hint::dynamic_quantization_group_size(ov::hint::dynamic_quantization::DISABLED));
  core.set_property(ov::hint::dynamic_quantization_group_size(ov::hint::dynamic_quantization::PER_TOKEN));

  core.set_property(ov::hint::dynamic_quantization::group_size(32));
  core.set_property(ov::hint::dynamic_quantization::group_size(ov::hint::dynamic_quantization::DISABLED));
  core.set_property(ov::hint::dynamic_quantization::group_size(ov::hint::dynamic_quantization::PER_TOKEN));
```

Reply:
OK I see. So we will have two hints for group_size. By the way, as this will require change from both CPU and GPU plugin, can I follow-up for this change in next PR? In this PR, I will just use a fixed value UINT64_MAX internally for per-token.

Reply:
I don't mind. But please revert current changes in the properties and than make PR with proper solution